lab-jonathan/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "lab-jonathan",
+  "version": "1.0.0",
+  "description": "lab08",
+  "main": "server.js",
+  "scripts": {
+    "test": "mocha",
+    "start": "node server.js"
+  },
+  "author": "JD",
+  "license": "ISC",
+  "dependencies": {
+    "bluebird": "^3.4.6",
+    "del": "^2.2.2",
+    "fetch": "^1.1.0",
+    "gulp": "^3.9.1",
```

Comment:
Fetch, Gulp, and SuperAgent are all testing modules, yes? Should then belong in dev dependencies.